spec/lib/iiif_manifest/v3/manifest_builder/placeholder_canvas_builder_spec.rb

```diff
@@ -17,6 +17,7 @@
     IIIFManifest::V3::DisplayContent.new(SecureRandom.uuid, type: 'Image',
                                                             width: 100,
                                                             height: 100,
+                                                            duration: 100,
                                                             format: 'image/jpeg')
   end
```

Comment:
Not a big issue but I think this might be confusing to have a duration with type Image for this example.  How about switching `type` and `format` for a video example or creating a separate test for Video content and leaving this one for Image content?